src/eduid/webapp/common/api/validation.py

```diff
@@ -58,7 +58,7 @@ def is_valid_password(password: str, user_info: Sequence[str], min_entropy: int,
     # Check password complexity with zxcvbn
     result = zxcvbn(password, user_inputs=user_info)
     _guesses = result.get("guesses", 1)
-    _pw_entropy = math.log(_guesses, 2)
+    _pw_entropy = math.log2(_guesses)
```

Comment:
log2(x) is usually more accurate than log(x, 2) according to
https://docs.python.org/3/library/math.html#math.log2
(Added in python 3.3)